scripts/install/setup_properties.sh

```diff
@@ -91,12 +93,15 @@ export HALYARD_VERSION=1.20.2
 
 # The specified network must exist, and it must not be a legacy network.
 # More info on legacy networks can be found here: https://cloud.google.com/vpc/docs/legacy
-export NETWORK=default
+export NETWORK=$NETWORK
 
 # If cluster does not exist, it will be created.
 export GKE_CLUSTER=${GKE_CLUSTER:-\$DEPLOYMENT_NAME}
 export ZONE=$ZONE
 export REGION=$REGION
+export SUBNET=$SUBNET
+export SPINNAKER_VERSION=1.14.5
```

Comment:
Looks like the these 2 lines (SPINNAKER_VERSION and HALYARD_VERSION) were accidentally duplicated. Please move the SUBNET assignment to be just below the NETWORK assignment.

Reply:
moved subnet by network and removed duplicated Spin & HAL version